posthog/test/__snapshots__/test_migration_0220.ambr

```diff
@@ -0,0 +1,93 @@
+# name: TagsTestCase.test_backfill_primary_dashboard
+  '
+  
+  SELECT c.relname,
+         CASE
+             WHEN c.relispartition THEN 'p'
+             WHEN c.relkind IN ('m',
+                                'v') THEN 'v'
+             ELSE 't'
+         END
+  FROM pg_catalog.pg_class c
+  LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
+  WHERE c.relkind IN ('f',
+                      'm',
+                      'p',
+                      'r',
+                      'v')
+    AND n.nspname NOT IN ('pg_catalog',
+                          'pg_toast')
+    AND pg_catalog.pg_table_is_visible(c.oid)
```

Comment:
I'm a bit confused by the SQL being generated by the snapshot here. I'm guessing this is Django migration sql, but I'm not positive.

Reply:
Also, it seems like django's bulk sql isn't captured in these snapshots.